### DISCUSSION TITLE
Incorrect "github repo info" in header

### DISCUSSION THREAD

**Discussion by buhtz** [mkdocs/mkdocs#3719] [Q&A]
![image](https://github.com/mkdocs/mkdocs/assets/11861496/21fd846c-3937-4163-bc61-45359e9e4d6e)

That field in the header indicate tag "v1.4.3". From a users perspective I would understand that the docu I do see currently is realted to that version/tag.

But that is not true. The docu generating that field is far beyond (~100 commits) that tag.

So, how is this field generated? Can I manipulate it somehow? Can I deactivate it?

Is there a way to indicate for real on which version (branch name, last commit, ...) that docu was build on?

**Comment by pawamoy** [comment]
That's a question for https://github.com/squidfunk/mkdocs-material.

The tag shown in the navbar is the latest GitHub release.

**Reply by buhtz** [reply]
I know that this is the latest release but it makes no sense because the doc wasn't build from that.

Follow-up: https://github.com/squidfunk/mkdocs-material/discussions/7223

**Reply by squidfunk** [reply]
The information is automatically extracted from the GitHub repository set in [`repo_url`](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#repository). If you set the URL to the repository of your documentation repository, the version from the latest release of this repository will be used, if any release exists. If no release exists, there will be no version tag.

Also see https://github.com/squidfunk/mkdocs-material/issues/2082#issuecomment-1790423790.

**Reply by pawamoy** [reply]
@buhtz you might also be interested in how to setup docs versioning with mike. See https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/?h=mike#versioning.
